src/meta/src/manager/catalog.rs

```diff
@@ -143,7 +143,11 @@ where
                 schema.upsert_in_transaction(&mut transaction)?;
                 schemas.push(schema);
             }
-            self.env.meta_store().txn(transaction).await?;
+            self.env
+                .meta_store()
+                .txn(transaction)
+                .await
+                .map_err(MetaError::transaction_error)?;
```

Comment:
Can we implement `From<MetaStoreError>` for `MetaError`?

Reply:
I don't recommend doing this. Because only `MetaStoreError` returned by `MetaStore::txn()` will be converted to `MetaError`.